widget/gonk/GeckoTouchDispatcher.h

```diff
@@ -81,6 +82,8 @@ class GeckoTouchDispatcher final
   TimeDuration mMaxPredict;      // How far into the future we're allowed to extrapolate
   TimeDuration mMinDelta;        // Minimal time difference between touches for resampling
 
+  bool mMouseDevice;
```

Comment:
I would prefer 'mMouseAvailable' for this and other related APIs. When this variable is true, it doesn't mean that we're notifying events because of mouse operations.  The cursor could also be moved when we're doing touch-move on a tablet when mouse plugged, right ?
Please also add a comment for this variable, thanks